RxExample/RxExample/Examples/GitHubSignup/DefaultImplementations.swift

```diff
@@ -105,9 +105,14 @@ class GitHubDefaultAPI : GitHubAPI {
     func signup(_ username: String, password: String) -> Observable<Bool> {
         // this is also just a mock
         let signupResult = arc4random() % 5 == 0 ? false : true
-        return Observable.just(signupResult)
-            .concat(Observable.never())
-            .throttle(0.4, scheduler: MainScheduler.instance)
+        
+        return Observable<Bool>.create { observable in
+            DispatchQueue.global().asyncAfter(deadline: DispatchTime.now() + 3) {
+                observable.onNext(signupResult)
+            }
+            return Disposables.create()
+            }
```

Comment:
Hi @gorozco58 ,

`delay` operator is here to simulate delay.

This code should be.

```
return Observable.just(signupResult)
     .delay(0.4, scheduler: MainScheduler.instance)
```

We don't need all of that other noise with dispatch queues.